addons/website_slides/models/slide_slide.py

```diff
@@ -126,16 +130,18 @@ class Slide(models.Model):
         ('infographic', 'Infographic'),
         ('presentation', 'Presentation'),
         ('document', 'Document'),
+        ('webpage', 'Web Page'),
         ('video', 'Video')],
         string='Type', required=True,
-        default='document',
+        default='document', readonly=True,
         help="The document type will be set automatically based on the document URL and properties (e.g. height and width for presentation and document).")
     index_content = fields.Text('Transcript')
     datas = fields.Binary('Content', attachment=True)
     url = fields.Char('Document URL', help="Youtube or Google Document URL")
     document_id = fields.Char('Document ID', help="Youtube or Google Document ID")
     link_ids = fields.One2many('slide.external.link', 'slide_id', string="External URL for this slide")
     mime_type = fields.Char('Mime-type')
+    html_content = fields.Html("HTML Content", help="Custom HTML content for slides of type 'Web Page'.")
```

Comment:
Shouldn't it be translatable ?

Reply:
Indeed, but note that the field is not displayed in backend. I don't know if it is possible to edit translation in frontend ....